packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -135,11 +155,13 @@ export const TestProviders = ({
                         fetchedToken={bypassAuth ? (isGuest ? guestToken : registerUserToken) : ''}
                     >
                         <CurrencyProvider currency={DEFAULT_CURRENCY}>
-                            <Router>
-                                <ChakraProvider theme={theme}>
-                                    <AddToCartModalProvider>{children}</AddToCartModalProvider>
-                                </ChakraProvider>
-                            </Router>
+                            <StoreLocatorProvider config={storeLocatorConfig}>
+                                <Router>
+                                    <ChakraProvider theme={theme}>
+                                        <AddToCartModalProvider>{children}</AddToCartModalProvider>
+                                    </ChakraProvider>
```

Comment:
Why is theme individually applied to components? Shouldn't the same theme apply to all pages and all components?

Reply:
@kzheng-sfdc this a test util, so this is just the work of mocking the theme provider which the chakra components need, but is absent since this is a unit test that doesn't include the `_app/index.jsx` provider contexts by default. At least that's my understanding, @kevinxh ?